pkgs/applications/audio/faust/faust2.nix

```diff
@@ -194,8 +196,8 @@ let
         # export parts of the build environment
         for script in "$out"/bin/*; do
           wrapProgram "$script" \
-            --set FAUSTLIB "${faust}/lib/faust" \
-            --set FAUST_LIB_PATH "${faust}/lib/faust" \
+            --set FAUSTLIB "${faust}/share/faust" \
```

Comment:
idk: it stopped working after the update, and when I looked, the needed files where in this path...
Should I file that as a bug upstream?